base/dates/arithmetic.jl

```diff
@@ -67,7 +67,7 @@ end
 (-)(y::Period,x::TimeType) = x - y
 
 for op in (:.+, :.-)
-    op_ = symbol(string(op)[2:end])
+    op_ = Symbol(string(op)[2])
```

Comment:
there are multi character dot operators that this loop may want to be extended to, I think this change is unnecessary and could make this bug prone later